### DISCUSSION TITLE
Quando usar Void

### DISCUSSION THREAD

**Discussion by klebertonlee** [balta-io/2801#83] [Perguntas e Respostas] [answered]
Quando usar void, não conseguir entender qual o momento e onde posso usar void.
poderia fazer mais um outro exemplo usando ele?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia @klebertonlee, como vai?

Todo método precisa retornar um valor, porém quando não queremos retornar nada, utilizamos a palavra `void` que significa **vazio** em inglês.
